mesonbuild/mesonmain.py

```diff
@@ -270,23 +266,32 @@ def run(original_args, mainfile):
             args = remaining_args
             cmd_name = args[0]
         if cmd_name == 'test':
+            from . import mtest
             return mtest.run(remaining_args)
         elif cmd_name == 'setup':
             args = remaining_args
             # FALLTHROUGH like it's 1972.
+        elif cmd_name == 'install':
+            from . import minstall
+            return minstall.run(remaining_args)
         elif cmd_name == 'introspect':
+            from . import mintro
             return mintro.run(remaining_args)
         elif cmd_name == 'rewrite':
+            from . import rewriter
             return rewriter.run(remaining_args)
         elif cmd_name == 'configure':
             try:
+                from . import mconf
                 return mconf.run(remaining_args)
             except MesonException as e:
                 mlog.exception(e)
                 sys.exit(1)
         elif cmd_name == 'wrap':
+            from .wrap import wraptool
```

Comment:
__[Flake8]__

[F811] redefinition of unused 'wraptool' from line 26


<!-- Comment by [SideCI](https://sideci.com) -->